lib/shopify_app/test_helpers/webhook_verification_helper.rb

```diff
@@ -0,0 +1,17 @@
+module ShopifyApp
+  module TestHelpers
+    module WebhookVerificationHelper
+      def authorized_webhook_verification_headers!
+        params = { foo: 'anything' }
```

Comment:
This should probably be a parameter passed in, in case they want to change the payload for any reason